charts/sonarqube-dce/values.yaml

```diff
@@ -69,7 +69,15 @@ searchNodes:
   # Set security context for sonarqube search container
   containerSecurityContext:
     # Sonarqube dockerfile creates sonarqube user as UID and GID 1000
+    # Those default are used to match pod security standard restricted as least privileged approach
```

Comment:
Can't we use YAML references to avoid duplication?

I'm particularly not very fond of it, but...

```yaml
defaultContainterSecurityContext: & defaultContainterSecurityContext
  allowPrivilegeEscalation: false
  runAsNonRoot: true
  runAsUser: 1000
  runAsGroup: 1000
  seccompProfile:
    type: RuntimeDefault
  capabilities:
    drop: ["ALL"]
...
  containerSecurityContext:
    <<: *defaultContainterSecurityContext
```

Reply:
Ha, that's a very good question. i think we should discuss that with Carmine, when we define how we want to structure and architecture the helm chart.

In the meantime, not found of it either, will keep as is even if quite verbose.